django/db/models/lookups.py

```diff
@@ -140,16 +135,8 @@ def as_oracle(self, compiler, connection):
         return lookup.as_sql(compiler, connection)
 
     @cached_property
-    def contains_aggregate(self):
-        return self.lhs.contains_aggregate or getattr(self.rhs, 'contains_aggregate', False)
-
-    @cached_property
-    def contains_over_clause(self):
-        return self.lhs.contains_over_clause or getattr(self.rhs, 'contains_over_clause', False)
-
-    @property
-    def is_summary(self):
-        return self.lhs.is_summary or getattr(self.rhs, 'is_summary', False)
```

Comment:
I think we should keep this property or add `c.is_summary = summarize` to the `resolve_expression()`.